src/lib/parts.ts

```diff
@@ -93,7 +164,11 @@ export class AttributeCommitter {
   commit(): void {
     if (this.dirty) {
       this.dirty = false;
-      this.element.setAttribute(this.name, this._getValue() as string);
+      let value = this._getValue();
+      if (sanitizeDOMValue) {
+        value = sanitizeDOMValue(value, this.name, 'attribute', this.element);
+      }
+      this.element.setAttribute(this.name, String(value));
```

Comment:
Remove the `String()` wrapper. `setAttribute()` already does toString coercion (the dom.d.ts typing is wrong).

Reply:
Needed for Symbol unfortunately.

```typescript
'' + Symbol(); // throws error
'' + String(Symbol())
```